modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
```

Comment:
```suggestion
== Disable LVMS volume snapshots
```
These aren't two things, it is one thing. The logical volume manager is the container storage interface.

Reply:
CSI driver and CSI snapshot are not the same. CSI provider or driver provides the interface to the storage system. CSI snapshot is a feature that allows managing snapshots of volumes. Both are implementations.

can be changed as "Disabling" CSI driver and CSI snapshot implementation"
disable is avoided based on the comment https://github.com/openshift/openshift-docs/pull/80823#discussion_r1740498227

Reply:
+1 this is correct. They are separate components with separate cluster workloads. If customers do not require snapshots, it can be disabled independent of the CSI driver.

Reply:
sentence is changed and updated as per discussion